engine-dmn/feel-scala/pom.xml

```diff
@@ -64,4 +68,30 @@
 
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <instructions>
+            <Import-Package>
+              !scala.*,
+              !org.camunda.feel.*,
```

Comment:
Would you mind explaining why you would want to exclude this from import? It is a required dependency of this module as far as I know, therefore I do not yet understand why it should be excluded here.